packages/renderer/noteStyle.ts

```diff
@@ -394,6 +398,7 @@ export default function(theme: any, options: Options = null) {
 
 		.mce-content-body .joplin-editable {
 			cursor: pointer !important;
+			${overflowHidden}
 		}
```

Comment:
I think `.mce-content-body` should apply to the TinyMCE rich text editor. Consider adding a comment explaining why this style is applied here.

Reply:
Fixed